src/myr/reference.js

```diff
@@ -253,6 +253,11 @@ let transformations = [
         description: <span>The setColor function changes the color of the cursor. For general colors, <a href='https://htmlcolorcodes.com/color-names/' target="_blank" rel="noopener noreferrer">HTML color codes</a> can be used (e.g., "blue", "deeppink"), and for specific colors HEX colors can be used (e.g., "#00ffff"). The default cursor state is "red".</span>,
         example: "setColor"
     },
+    {
+        name: "setTexture",
+        parameters: [{type: "string", name: "texture"}],
+        description: <span>The setTexture function changes the texture of the cursor. The unaltered texture is displayed when the color is white, otherwise the texture will be affected by the current color. Textures can either be applied using the name of one of these /*insert link here*/ colors (e.g. "brick") or by inserting a valid url (e.g. "https://upload.wikimedia.org/wikipedia/commons/5/5e/Water_drop_001.jpg")</span>,
+    },
```

Comment:
What about getTexture?

Reply:
No other getters are currently documented in the reference, so I did not document getTexture. would you like me to create references to the other getters along with getTexture while im here?